Jellyfin.Plugin.ThePornDB/Providers/MetadataAPI.cs

```diff
@@ -191,7 +191,7 @@ public static async Task<MetadataResult<Movie>> SceneUpdate(string sceneID, stri
                             break;
                     }
 
-                    if (!string.IsNullOrEmpty(curID))
+                    if (!string.IsNullOrEmpty(curID) && !Plugin.Instance.Configuration.DisableActorsAutoIdentify)
```

Comment:
Still not agree with that, this is 100% info. Idea was not pick automatically first search result, cause it can be wrong